README.md

```diff
@@ -1,43 +1,56 @@
 
 ### Source Installation
+When running on Linux or developing your own plugins, the most straight forward method is to copy the plugin sources to your Cellprofiler plugins directory. It is recommended to use the default ``CellProfiler\plugins`` folder as your plugin directory, but you can still have the plugins in other locations. Follow the instructions for installing CellProfiler on the [Wiki](https://github.com/CellProfiler/CellProfiler/wiki). Choose you operating system on the right side panel. Once you have installed CellProfiler, set the plugins directory in Cellprofiler preferences, then save and restart CellProfiler. 
+
+Some useful guidlines for developing CellProfiler plugins:
+
+[Orientation to CellProfiler code](https://github.com/CellProfiler/CellProfiler/wiki/Orientation-to-CellProfiler-code)
+
+[CellProfiler 3 to 4 migration guide](https://github.com/CellProfiler/CellProfiler-plugins/wiki/CellProfiler-3-to-4-migration-guide)
 
-When running on Linux or developing your own plugins, the most straight forward method is to copy the plugin sources to your Cellprofiler plugins directory.
+### Previous Plugins
+Please note [Previous plugins](https://github.com/sa2c/CellProfiler-RunOnCluster/archive/master.zip) do not fully work with the precompiled CellProfile executable (version 3.1.8), which has default settings for the SCW Sunbird cluster, can be found in [releases](https://github.com/sa2c/CellProfiler-RunOnCluster/releases/download/v1.0/CellProfiler.exe). These executables require that a Java Runtime Environment is installed. You can get one for example form [Java](https://www.java.com/en/download/).
 
-Follow the instructions for installing CellProfiler on the [wiki](https://github.com/CellProfiler/CellProfiler/wiki). Choose you operating system on the right side panel. Once you have installed CellProfiler, set the plugins directory in Cellprofiler preferences. Download [the plugins](https://github.com/sa2c/CellProfiler-RunOnCluster/archive/master.zip) and copy move the files to your plugins directory.
+### Latest Plugins
+Please download the [plugins](https://codeload.github.com/sa2c/CellProfiler-RunOnCluster/zip/refs/heads/cellprofiler-4.2.2rc) to your plugins directory. In the plugins directory, install the requirements for the plugins:
 
-In the plugins directory, install the additional requirements for the plugins:
 ```
-python2 -m pip install -r requirements.txt
+python -m pip install -r requirements.txt
 ```
 
-## Usage
+Please be aware that if `pip install` does not install the `Rynner` or the `libsubmit` library properly (i.e. there is no ``Rynner`` folder inside ``site-packages``.). You should copy the `Rynner` folder in this repository and paste it inside ``side-packages``. You may also need to copy the `libsubmit` folder in this branch to ``side-packages``.
```

Comment:
This isn't an acceptable user experience. There should be a work item opened against this to be fixed.

Reply:
Fixed the `rynner` installation issue.